const.go

```diff
@@ -111,7 +125,29 @@ var ConfToEnv = map[string]string{
 	// [Databases] -- not supported
 
 	// [MessageQueue] -- core-data only
-	"messagequeue.topic": "core-data/MESSAGEQUEUE_TOPIC",
+	// Indicates the message bus implementation to use, i.e. zero, mqtt, redisstreams...
+	"messagequeue.type": "core-data/MESSAGEQUEUE_TYPE",
```

Comment:
Is `[MessageQueue]` also applicable to app-service-configurable, and device-virtual? If so, maybe we should make the prefix a csv list?

Reply:
After further review, maybe app-service-configurable just uses `[Binding]`?

Reply:
It's in device-virtual (https://github.com/edgexfoundry/device-virtual-go/blob/12c6854996096b74e90d43960f739a4a5ae0269d/cmd/res/configuration.toml#L37) but not in ASC. ASC now uses Trigger and [Trigger.EdgexMessageBus] instead of Binding - see https://github.com/edgexfoundry/app-service-configurable/blob/01778f3054f56a5e1c9fa7a64eee47acbae495e2/res/mqtt-export/configuration.toml#L129

Reply:
OK, thanks for clarifying. Since it's used by device-virtual, then I think we have two choices:

 - remove the `core-data` prefix, which means any service could set these OR
 - update the validation logic to treat the prefix as a csv, and add `,device-virtual` to the prefix

Reply:
Adding csv support is a good idea. I'll push an update with that